frontend/packages/kubevirt-plugin/integration-tests/tests/vm.template.wizard.scenario.ts

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-undef, max-nested-callbacks */
+import { OrderedMap } from 'immutable';
+import { testName } from '../../../../integration-tests/protractor.conf';
+import {
+  removeLeakedResources,
+  createResources,
+  deleteResources,
+  addLeakableResource,
+} from '../../../console-shared/src/test-utils/utils';
+import { NetworkResource, StorageResource, ProvisionOption } from './utils/types';
+import { VM_BOOTUP_TIMEOUT_SECS } from './utils/consts';
+import {
+  basicVmConfig,
+  rootDisk,
+  networkInterface,
+  multusNad,
+  hddDisk,
+} from './utils/mocks';
+import {
+  getTestDataVolume,
+} from './vm.wizard.configs';
+import { VirtualMachine } from './models/virtualMachine';
+import { VirtualMachineTemplate } from './models/virtualMachineTemplate';
+
+describe('Kubevirt create VM Template using wizard', () => {
+  const leakedResources = new Set<string>();
+  const testDataVolume = getTestDataVolume(testName);
+  const commonSettings = {
+    startOnCreation: true,
```

Comment:
template config should not include startOnCreation

Reply:
:+1: moved to the `vmConfig` const